.github/workflows/incident_report.yml

```diff
@@ -0,0 +1,19 @@
+# This workflow assigns to every issue (re)opened or labeled with label type/incident the Incident Report project.
+
+name: Project Incident assignment
+
+on:
+  issues:
+    types:
+      - opened
+      - reopened
+      - labeled
+
+jobs:
+  incident-report:
+    if: ${{ github.event.label.name == 'type/incident' }}
```

Comment:
I don't think this will work: what you did was a good solution for a labelling event, ie when you add a label to an issue (like documented [here](https://docs.github.com/en/actions/managing-issues-and-pull-requests/commenting-on-an-issue-when-a-label-is-added)) but you have 2 cases: labelling event or issue (re-)opening, and in the 2nd case this will likely not work (for instance you can have several labels). 

We need to
- add a solution like [this one](https://stackoverflow.com/questions/59588605/how-to-check-for-a-label-in-a-github-action-condition) in the mix, but then we need to be smart on the if clause
- simplify this to work only on labelling event ; I don't think this will work when an issue is created with a label already, but we never know